### PR TITLE
notice api permissions about mirror pods

### PR DESCRIPTION
**Pull request by CaoShuFeng**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.11 Features: set Milestone to 1.11 and Base Branch to release-1.11
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> NOTE: After opening the PR, please *un-check and re-check* the ["Allow edits from maintainers"](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) box so that maintainers can work on your patch and speed up the review process. This is a temporary workaround to address a known issue with GitHub.> 
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/static-pod.md [hunk 1]
```diff
@@ -89,7 +89,8 @@ static-web-my-node1        1/1       Running   0          2m
 
 Labels from the static pod are propagated into the mirror-pod and can be used as usual for filtering.
 
-Notice we cannot delete the pod with the API server (e.g. via [`kubectl`](/docs/user-guide/kubectl/) command), kubelet simply won't remove it.
+Note that we should make sure kubelet have enough permissions to create the mirror pod in API server, otherwise the creation request may be forbidden by API server(e.g. [PodSecurityPolicy](/docs/concepts/policy/pod-security-policy/)).
```
Comment (ajaeger) [inline content/en/docs/tasks/administer-cluster/static-pod.md]: Nit: Missing space (" ") before the opening "(", it should be "API server (e.g....)"

Comment (CaoShuFeng) [reply content/en/docs/tasks/administer-cluster/static-pod.md]: Done.

**Comment by mistyhacks** [comment]
I added a copyedit. PTAL. I am not 💯 that my rewording on the last sentence is accurate, so review is appreciated.

File: content/en/docs/tasks/administer-cluster/static-pod.md [hunk 2]
```diff
@@ -89,8 +89,13 @@ static-web-my-node1        1/1       Running   0          2m
...
 Labels from the static pod are propagated into the mirror-pod and can be used as usual for filtering.
 
-Note that we should make sure kubelet have enough permissions to create the mirror pod in API server, otherwise the creation request may be forbidden by API server (e.g. [PodSecurityPolicy](/docs/concepts/policy/pod-security-policy/)).
-Also we cannot delete the pod with the API server (e.g. via [`kubectl`](/docs/user-guide/kubectl/) command), kubelet simply won't remove it.
+{{< note >}}
+**Note**: Make sure the Kubelet have permission to create the mirror pod in the API server.
+If not, the creation request is forbidden by the API server. See
+[PodSecurityPolicy](/docs/concepts/policy/pod-security-policy/). A permission error also
+causes the API server to fail to delete the pod. If you try, such as by using kubectl, the
+Kubelet cannot remove it.
```
Comment (CaoShuFeng) [inline content/en/docs/tasks/administer-cluster/static-pod.md]: Sorry, I should split these two sentences.
In fact the `delete` description has nothing to do with permissions.
It means even if you delete the mirror pod with `kubectl delete`, the kubelet will create it again.

Comment (tengqm) [reply content/en/docs/tasks/administer-cluster/static-pod.md]: > If not, the creation request is forbidden by the API server.

This sentence can be removed. Or you can change "forbidden" to "rejected".

> If you try, such as by using kubectl, the Kubelet cannot remove it.

This sentence can be removed.

Comment (CaoShuFeng) [reply content/en/docs/tasks/administer-cluster/static-pod.md]: @tengqm 
updated.

**Comment by mistyhacks** [comment]
This whole topic could use some copyediting but it's out of scope for this PR. Thanks for the contribution!

/approve
/lgtm

**Pull request merged**